observable/observable_test.go

```diff
@@ -647,3 +647,99 @@ func TestObservableScanWithString(t *testing.T) {
 
 	assert.Exactly(t, expected, words)
 }
+
+func TestRepeateInfinityOperator(t *testing.T) {
+	myStream := Repeat("mystring")
+
+	item, err := myStream.Next()
+
+	if err != nil {
+		assert.Fail(t, "fail to emit next item", err)
+	}
+
+	if value, ok := item.(string); ok {
+		assert.Equal(t, value, "mystring")
+	} else {
+		assert.Fail(t, "fail to emit next item", err)
+	}
+}
+
+func TestRepeateNtimeOperator(t *testing.T) {
+	myStream := Repeat("mystring", 2)
+	stringarray := []string{}
+
+	onNext := handlers.NextFunc(func(item interface{}) {
+		if value, ok := item.(string); ok {
+			stringarray = append(stringarray, value)
+		}
+	})
+
+	onDone := handlers.DoneFunc(func() {
+		stringarray = append(stringarray, "end")
+	})
+
+	sub := myStream.Subscribe(observer.New(onNext, onDone))
+	<-sub
+
+	assert.Exactly(t, []string{"mystring", "mystring", "end"}, stringarray)
+}
+
+func TestRepeateNtimeMultiVariadicOperator(t *testing.T) {
+	myStream := Repeat("mystring", 2, 2, 3, 4, 5, 6, 7)
+	stringarray := []string{}
+
+	onNext := handlers.NextFunc(func(item interface{}) {
+		if value, ok := item.(string); ok {
+			stringarray = append(stringarray, value)
+		}
+	})
+
+	onDone := handlers.DoneFunc(func() {
+		stringarray = append(stringarray, "end")
+	})
+
+	sub := myStream.Subscribe(observer.New(onNext, onDone))
+	<-sub
+
+	assert.Exactly(t, []string{"mystring", "mystring", "end"}, stringarray)
+}
+
+func TestRepeateWith0TimesOperator(t *testing.T) {
```

Comment:
Extra 'e' typo and please change from using `0` in the function name to `Zero` instead.